doc/util/literate_chapel.py

```diff
@@ -102,21 +107,25 @@ def push_line(line):
                 # No need for trailing white space... ever
                 proseline = proseline.rstrip(' ')
 
-            # Handle indentation
```

Comment:
nothing that `literate_chapel.py` is also used for the blog so it would be good to make sure that none of the blog Markdown conversions are affected.

Reply:
Can you suggest how I can go about doing that?

Reply:
I think you would:

1. Follow the steps in the README to build the `main` branch of `chapel-blog` with `CHPL_HOME` pointed to a version of the repo right before this PR
  * This would involve running `./scripts/chpl_blog.py build`
2. Preserve the `public` directory by renaming it to something (`public-old`)
3. Going back to the Chapel repo at `$CHPL_HOME`, check out this PR; run `build` again
4. Do a `diff` on `public-old` and `old`.

Reply:
There are whitespace changes but I think they're innocuous and the new result is more accurate.

An example, with this input within a comment:

````
/*
### First Solution: Hand-coded Interval Arithmetic
Here is a succinct solution for both parts in Chapel.

```Chapel
  use IO;

  var sumSubset = 0;
...
````

Now there are spaces in the visible code listing before things like `use`.

This problem only affects 2 posts: aoc2022-day04-ranges (source of the above example) and netcdf1/index.html (which has a minor difference that seems to be an improvement).